examples/loader-google-bigquery/src/data/covidstats_it.csv.js

```diff
@@ -0,0 +1,54 @@
+import { csvFormat, csvParse } from "d3-dsv";
+import { runQuery } from "./google-bigquery.js";
+import { promises as fs } from "fs";
+import { dirname, join } from "path";
+import { fileURLToPath } from "url";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const query = `
+  SELECT 
+    FORMAT_TIMESTAMP('%Y-%m-%d', date) as formatted_date, 
```

Comment:
I think this should be called `date` throughout.

Reply:
Yes, can be updated if needed